project/target/auto-grader-1/scripts/gradeSubmissions.js

```diff
@@ -0,0 +1,29 @@
+async function getOptions() {
+    const assignmentKey = getUrlVars()['assignment-key'];                                                                                                                                                                       
+    var myDiv = document.getElementById('grading-options');
+
+    var gradeIndivid = document.createElement('a');
+    gradeIndivid.innerHTML = "Grade Individual Submissions<br><br>";
```

Comment:
Just a note check out using .innerText instead to see if you get the same results.

Reply:
Sorry -- outdated /target code